src/ucp/tag/tag_send.c

```diff
@@ -222,6 +221,8 @@ static void ucp_tag_send_req_init(ucp_request_t* req, ucp_ep_h ep,
     req->send.tag          = tag;
     req->send.reg_rsc      = UCP_NULL_RESOURCE;
     req->send.state.offset = 0;
+    req->send.uct_comp.count = 0;
```

Comment:
why move it here instead of tag_send.c:112? it could be done only for zcopy requests